### PR TITLE
Chores/application edit fields update #158821830

### PR DESCRIPTION
**Pull request by fddayan**
https://www.pivotaltracker.com/story/show/158821830

Merge https://github.com/Exygy/sf-dahlia-lap/pull/38 before merging this PR.

### CODE REVIEW

**Comment by chrisdolendo** [comment]
@fddayan did you do a q-a test on the review app for this chore? I'm getting an error when trying to open the applications page:

`NoMethodError (undefined method `parse_results_for_fields' for #<Force::ApplicationService:0x000000027047f0>):`

**Review by chrisdolendo — CHANGES_REQUESTED** [review]
cannot open applications page on review app :(

**Comment by fddayan** [comment]
@chrisdolendo I merged with master, maybe something got broken let me check

**Comment by fddayan** [comment]
@chrisdolendo you can test now. Fixed merging problems

**Comment by chrisdolendo** [comment]
@fddayan 
The following are error-ing and don't error on Full :( 
https://dahlia-lap-full-pr-43.herokuapp.com/listings/lease_ups/a0W0P00000DZfSpUAL/applications
https://dahlia-lap-full-pr-43.herokuapp.com/listings/a0W0P00000F8YG4UAN

can you also look into this code on ApplicationPage.js?
```
const buildActionLinkIfNecessary = (app) => {
  if (!app.isLotterComplete && app.submissionType === 'Paper') {
    return { title: 'Edit Application', link: appPaths.toApplicationEdit(app.id) }
  } else {
    return undefined
  }
}
```

- `isLotterComplete` is a typo (across all the app) - should be `isLotteryComplete`
- `applicationSubmissionType` should be `application_submission_type`

Can you check on this again and do a smoke test on the features of the app before re-review?

**Comment by fddayan** [comment]
@chrisdolendo ready

File: app/javascript/components/applications/applicationsPageMapper.js [hunk 1]
```diff
@@ -1,52 +1,52 @@
...
-
-const mapProperties = ({ listings }) => {
-  return {
-    onFetchData: onFetchData,
-    listings: map(listings, mapListingsOptions)
-  }
-}
-
-export default mapProperties
+// import { map } from 'lodash'
```
Comment (callaghanc) [inline app/javascript/components/applications/applicationsPageMapper.js]: @fddayan what is the purpose of this commented out file?

File: spec/javascript/components/applications/application_form/PaperApplicationForm.test.js.old [hunk 2]
```diff
@@ -0,0 +1,56 @@
+// import React from 'react'
```
Comment (callaghanc) [inline spec/javascript/components/applications/application_form/PaperApplicationForm.test.js.old]: @fddayan what is the purpose of this commented out file?

**Review by chrisdolendo — APPROVED** [review]
After several reviews and re-reviews, this PR is ready to go! :+1: YAY!

**Pull request merged**